src/messages/MMgrBeacon.h

```diff
@@ -59,12 +59,12 @@ class MMgrBeacon final : public PaxosServiceMessage {
              entity_addrvec_t server_addrs_, bool available_,
 	     std::vector<MgrMap::ModuleInfo>&& modules_,
 	     std::map<std::string,std::string>&& metadata_,
-             std::vector<entity_addrvec_t> clients,
+	     std::multimap<std::string, entity_addrvec_t>&& clients_,
```

Comment:
Unrelated, referring to the whitespace noise at the beginning of the line.  I think we really need to stop doing this and drop the "compress 8 spaces to a single tab character" bit from our amendments to Google's style guide:
https://github.com/ceph/ceph/blob/e70c55cb7bd583b072ffa3dbeac123cf4021b8cf/CodingStyle#L77-L78
It has never been followed consistently across the project and mixing tabs and spaces this way (when the base indent is 2 spaces, not a tab!) is just bad.

Reply:
Oops! I will correct this.